docs/devguide/repl.rst

```diff
@@ -7,16 +7,25 @@ Accessing the REPL
 REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
 in real-time as you write it.
 
-Accessing the REPL on the micro:bit will require you to:
+.. note::
 
-* Determine the communication port identifier for the micro:bit
-* Use a program to establish communication with the device
+    For Windows 7 you might need to install the 
+    `MBED serial driver <https://os.mbed.com/docs/mbed-os/v6.8/program-setup/windows-serial-driver.html>`_.
```

Comment:
I'd removed the Mbed serial info as it is no longer needed and can affect WebUSB

Reply:
Done.